From a1c14cce4121862bb12fe34f294b5ab1d3dce7da Mon Sep 17 00:00:00 2001
From: Vincent Lefevre <vincent@vinc17.net>
Date: Tue, 8 Oct 2019 17:06:57 +0200
Subject: [PATCH] Update for Debian 8.0 (jessie), 9.0 (stretch) and 10.0
 (buster).

Also removed the obsolete debian_version and debian_release lines,
as Distribution.pm no longer supports multiple release files and
they made the tests fail (fixes issue #2 for Debian).

Bug: https://github.com/burak/CPAN-Sys-Info-Driver-Linux/pull/3

Added bullseye and bookworm as well.

---
 lib/Sys/Info/Driver/Linux/OS/Distribution/Conf.pm | 8 ++++++--
 1 file changed, 6 insertions(+), 2 deletions(-)

--- a/lib/Sys/Info/Driver/Linux/OS/Distribution/Conf.pm
+++ b/lib/Sys/Info/Driver/Linux/OS/Distribution/Conf.pm
@@ -31,8 +31,6 @@
 <debian>
     manufacturer  = Debian Project
     version_match = (.+)
-    release = debian_version
-    release = debian_release
     <edition>
            1.1  = buzz
            1.2  = rex
@@ -46,21 +44,31 @@
            5.0  = lenny
            6.0  = squeeze
            7.0  = wheezy
+           8.0  = jessie
+           9.0  = stretch
+          10.0  = buster
+          11.0  = bullseye
+          12.0  = bookworm
     </edition>
     # we get the version as "lenny/sid" for example
     <vfix>
-        buzz    = 1.1
-        rex     = 1.2
-        bo      = 1.3
-        hamm    = 2.0
-        slink   = 2.1
-        potato  = 2.2
-        woody   = 3.0
-        sarge   = 3.1
-        etch    = 4.0
-        lenny   = 5.0
-        squeeze = 6.0
-        wheezy  = 7.0
+        buzz     = 1.1
+        rex      = 1.2
+        bo       = 1.3
+        hamm     = 2.0
+        slink    = 2.1
+        potato   = 2.2
+        woody    = 3.0
+        sarge    = 3.1
+        etch     = 4.0
+        lenny    = 5.0
+        squeeze  = 6.0
+        wheezy   = 7.0
+        jessie   = 8.0
+        stretch  = 9.0
+        buster   = 10.0
+        bullseye = 11.0
+        bookworm = 12.0
     </vfix>
 </debian>
 
